inst/templates/bioc-issue.md

```diff
@@ -1,56 +1,53 @@
 ---
-name: Bug report or feature request
-about: Describe a bug you've seen or make a case for a new feature
-title: "[BUG] Your bug or feature request"
+name: Bug Report
+about: Describe the bug in detail
+title: "[BUG] A short description of the bug"
 labels: ''
 assignees: ''
 ---
 
-Please briefly describe your problem and what output you expect. If you have a question, please don't use this form. Instead, ask on <https://support.bioconductor.org/> using the appropriate tag(s) including one for this package.
+Please ask questions about how to use `{{{Package}}}` on the Bioconductor
+Support Site at <https://support.bioconductor.org> using the appropriate tag(s)
+including the one for this package.
 
-## Context
+**Note**. Update the issue title to concisely describe the bug.
 
-Provide some context for your bug report or feature request. This could be the:
+## Describe the bug
 
-* link to raw code, example: https://github.com/lcolladotor/osca_LIIGH_UNAM_2020/blob/master/00-template.Rmd#L24-L28
-* link to a commit, example: https://github.com/lcolladotor/osca_LIIGH_UNAM_2020/commit/6aa30b22eda614d932c12997ba611ba582c435d7
-* link to a line of code inside a commit, example: https://github.com/lcolladotor/osca_LIIGH_UNAM_2020/commit/6aa30b22eda614d932c12997ba611ba582c435d7#diff-e265269fe4f17929940e81341b92b116R17
-* link to code from an R package, example: https://github.com/LieberInstitute/spatialLIBD/blob/master/R/run_app.R#L51-L55
+Please provide a clear and concise description of what the bug is.
 
-## Code
+### Provide a minimally reproducible example (reprex)
 
-Include the code you ran and comments
+Provide a clear and concise description of the bug that is best illustrated
+with a minimally reproducible example using
+[`reprex::reprex()`](https://reprex.tidyverse.org/reference/reprex.html).
+For tips, see this link on Stackoverflow:
+<https://stackoverflow.com/questions/5963269/how-to-make-a-great-r-reproducible-example>
 
-```R
-## prompt an error
-stop('hola')
+## Expected behavior
 
-## check the error trace
-traceback()
-```
+A clear and concise description of what you expected to happen.
 
-## Small reproducible example
+## R Session Information
 
-If you copy the lines of code that lead to your error, you can then run [`reprex::reprex()`](https://reprex.tidyverse.org/reference/reprex.html) which will create a small website with code you can then easily copy-paste here in a way that will be easy to work with later on.
+Please report the output of either `sessionInfo()` or `session_info()` here.
 
 ```R
-## prompt an error
-stop('hola')
-#> Error in eval(expr, envir, enclos): hola
-
-## check the error trace
-traceback()
-#> No traceback available
+options(width = 120)
+## insert session info here
 ```
 
+Note. The output of `sessioninfo::session_info()` includes relevant GitHub
```

Comment:
I like the tone of this -- `sessionInfo()` is OK, but `session_info()` might be better. But on the other hand details of GitHub installation shouldn't be relevant for debugging Bioconductor packages.

It would be very useful to add `BiocManager::valid()` since mixed package installations can be a major source of headache both for the user and developer.